Regression/Checksum/checksum.py

```diff
@@ -48,39 +44,27 @@ def read_plotfile(self, do_fields=True, do_particles=True):
         @param do_particles Whether to read particles from the plotfile.
         '''
 
-        ds = yt.load(self.plotfile)
-        grid_fields = [item for item in ds.field_list if item[0] == 'boxlib']
-        species_list = set([item[0] for item in ds.field_list if
-                            item[1][:9] == 'particle_' and item[0] != 'all'])
+        ds = Backend(self.plotfile)
+        grid_fields = ds.fields_list()
```

Comment:
Looking at the lines that were removed, maybe the correct name was `ds.field_list`? (the error on CI in fact says that "name 'grid_fields' is not defined")
```suggestion
        grid_fields = ds.field_list()
```